nf_core/modules/lint/main_nf.py

```diff
@@ -108,7 +108,7 @@ def check_script_section(self, lines):
 
     # check for prefix (only if module has a meta map as input)
     if self.has_meta:
-        if re.search("\s*prefix\s*=\s*options.suffix", script):
+        if re.search("\s*prefix\s*=\s*task.args.prefix", script):
```

Comment:
this should be `task.ext`? 

And are you sure you are creating your  PR off the latest `dev` branch? `options.suffix` should not be there anymore. (Could be that we forgot, but then the linting should have failed :thinking: )

Reply:
Oops, good catch!

I'm pretty sure... but I can try again.

Reply:
@grst yup, the `options.suffix` is still in current `dev`: https://github.com/jfy133/nf-core-tools/blob/7d89ec3bc3faac50dfb8307b00d369b5ed3006dd/nf_core/modules/lint/main_nf.py#L109-L114